server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxGetMethodContract.scala

```diff
@@ -458,61 +291,6 @@ trait MailboxGetMethodContract {
          |}""".stripMargin)
   }
 
-  @Test
-  def getMailboxesShouldReturnAllPropertiesWhenNotSupplied(server: GuiceJamesServer): Unit = {
```

Comment:
You can rewrite this test by doing a get on a single mailbox.

Reply:
I deleted this one because the test by doing a get on a single mailbox existed already, with the test `getMailboxesByIdsShouldReturnCorrespondingMailboxWithoutPropertiesFromNotProvidedCapabilities` doing pretty much that I believe.

Useless to have duplicates :))